lib/faexport/scraper.rb

```diff
@@ -489,7 +489,228 @@ def submit_journal(title, description)
     }
   end
 
+  def new_submissions(from_id)
+    # Set pagination
+    url = "msg/submissions/new"
+    if from_id
+      url << "~#{from_id}@72/"
+    end
+
+    # Get page code
+    html = fetch(url)
+
+    login_user = get_current_user(html)
+    submissions = html.css('.gallery > figure').map{|art| build_submission_notification(art)}
+    {
+        "current_user": login_user,
+        "new_submissions": submissions
+    }
+  end
+
+  def notifications(include_deleted)
+    # Get page code
+    html = fetch("msg/others/")
+    # Parse page
+    login_user = get_current_user(html)
+    # Parse new watcher notifications
+    new_watches = []
+    if html.at_css("ul#watches")
+      html.at_css("ul#watches").css("li:not(.section-controls)").each do |elem|
+        if elem.at_css("input")['checked'] == "checked"
+          if include_deleted
+            new_watches << {
+                watch_id: "",
+                name: "Removed by the user",
+                profile: fa_address,
```

Comment:
I would just leave this blank rather than inserting a default address

Reply:
Alright, done that in effbdcec01e83d25bf6c81facb65d7c8650b24d9